### DISCUSSION TITLE
FindShops API to support filters for sort_on and sort_order

### DISCUSSION THREAD

**Discussion by switmac** [etsy/open-api#657] [Ideas]
https://developer.etsy.com/documentation/reference/#operation/findShops

FindShops API to support filters for sort_on (created_date) and sort_order(asc, desc)
something like
![image](https://user-images.githubusercontent.com/3896300/201263864-fd082b6d-625b-4ed8-b773-ec5cd7ab9610.png)

We would like to get the latest created Shops in Etsy

in v2 we are able to achieve this by using `https://openapi.etsy.com/v2/shops/?limit=100&offset=$offset` as the response is returning the `latest created shops by default`

however in v3, the response of https://openapi.etsy.com/v3/application/shops 
the shops returned here are from created 17 and 15 yrs ago

![image](https://user-images.githubusercontent.com/3896300/201275663-a2636aff-a8f4-4f65-a73f-aecf5e591f10.png)

we tried to stretch out using the limit and offset filters but the limit for offset is only up to `10000` 
and then it shows use this error
![image](https://user-images.githubusercontent.com/3896300/201263717-5e0c61c8-9ebb-4269-8ecd-deff0c001394.png)

**Comment by etsyachristensen** [comment]
@switmac The issue is that the findShops endpoint is meant to be a search. So by not providing a shop name, it's essentially searching all shops and returning in ascending order of relevance. However, because there's no string in the shop name there's nothing for the system to judge relevance.

In OpenAPI, we do not have an endpoint that will simply return all shops sorted by any specific metric.

**Reply by switmac** [reply]
noted on this @etsyachristensen .. thank you!

**Comment by emergingdzns** [comment]
@nateswart and @jasonpincin We are desperately in need of a way to use the findShops endpoint but WITHOUT requiring the name, but still being able to sort by created date desc. Essentially what we need is a paginated recent shops option. Not sure if that means a new endpoint or change to the current endpoint. Can you help us out. It's one of the things keeping us from migrating to V3.
